src/services/coreService.ts

```diff
@@ -63,8 +72,16 @@ const deleteTaskById = async function(id: number, options?: object): Promise<nul
 	return axios.delete(generateOcsUrl('taskprocessing/task/{id}', { id }), options)
 }
 
+const searchMessages = async function(params: SearchMessagePayload, options: object) {
+	return axios.get(generateOcsUrl('search/providers/talk-message/search'), {
```

Comment:
We can move to search in current room only (see lib/Search/CurrentMessageSearch.php)
```suggestion
	return axios.get(generateOcsUrl('search/providers/talk-message-current/search'), {
```

Payload should contain `from: '/call/${token.value}',` instead of conversation